PRS7001-8000/PRS7597Nob.xml

```diff
@@ -54,11 +54,16 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <persName xml:lang="gez" type="normalized" corresp="#n1">Nob</persName>
                <persName xml:lang="gez" xml:id="n2">ኖብ፡ ወልደ፡ ያሳይ፡</persName>
                <persName xml:lang="gez" type="normalized" corresp="#n2">Nob Walda Yāsāy</persName>
-               <floruit>He was the brother of ʾaṣe <persName ref="PRS10342Yeshaq">Yǝsḥāq</persName> and spiritual son of <persName ref="PRS10253Yasay">Yāsāy</persName>.
-             He is credited as the founder of the monastic community of <ref type="ins" corresp="INS0359Rema">Remā</ref>.</floruit>
+               <persName xml:lang="gez" type="normalized"><roleName type="title">ʾabbā</roleName></persName>
+               <occupation type="ecclesiastic">He is credited as the founder of the monastic community of 
+                  <ref type="ins" corresp="INS0359Rema">Remā</ref>.</occupation>
+               <floruit notBefore="1350" notAfter="1450">Mid 14th to mid 15th century</floruit>
             </person>
             <listRelation>
-               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"/>
+               <relation name="snap:BrotherOf" active="PRS7597Nob" passive="PRS10342Yeshaq"><desc>He was the brother 
```

Comment:
note that the descriptions of relations are only visible from the tab relations and not from the main record; i think the spiritual relation is too important to be "hidden" this way

Reply:
OK. I will move it back to floruit. Probably, it is useful to have it both in floruit and in relation.